recipes/sentry-native/all/conandata.yml

```diff
@@ -1,13 +1,13 @@
 sources:
+  "0.7.17":
+    url: "https://github.com/getsentry/sentry-native/releases/download/0.7.17/sentry-native.zip"
+    sha256: "c1341a0ac02440db65f41b968a46979ceab8de765c2407efb61a99511346e098"
+  "0.7.16":
```

Comment:
thanks for your PR @MartinDelille  -  is there a specific reason why 0.7.16 may be needed and someone wouldn't use the most recent version available? unless there is a strong reason, please update the PR to only add the most recent version, thanks!